examples/camel-example-jooq/src/main/resources/log4j.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@DenisIstomin , is it possible to use common for all components/examples log4j2.properties config instead of new one log4j.xml?